app/models/patient_phone_number.rb

```diff
@@ -54,4 +54,10 @@ def update_whitelist_requested_at(time)
 
     exotel_phone_number_detail.update(whitelist_requested_at: time)
   end
+
+  def number_with_country_code
```

Comment:
I think this should probably be happening as a validation on the model and "normalizing" the number on save. We should also probably not be overriding the country code is already present, but this PR is not intended to change existing functionality.

Reply:
I agree we should move this to the model eventually, but I think there is enough bad data out there w/ the phone numbers to make that premature right now.